docs/project-docs/developer-guide.md

```diff
@@ -66,3 +66,15 @@ One can build in Debug or Release mode from the root by doing `build.cmd -Releas
 
 We only support 64-bit binaries right now.
 
+### Updating manifest and ep-list files
+
+During development, there may arise a need to update the current baseline `core_manifest.json` and/or `core_ep-list.tsv` files. For example, a change in the name or type of a variable in a given class in the API that is not reflected in `core_manifest.json` will trigger the following failure:
+
+`*** Failure: Output and baseline mismatch at line 123 , expected ' ...x... ' but got ' ...y...' : '../Common/EntryPoints/core_manifest.json'`
+
+Steps to update `core_manifest.json` and `core_ep-list.tsv`:
+1. Unskip the `RegenerateEntryPointCatalog` unit test in `test/Microsoft.ML.Core.Tests/UnitTests/TestEntryPoints.cs`. This can be done by temporarily commenting out the skip attribute on the unit test for `RegenerateEntryPointCatalog` (`[Fact(Skip = "Execute this test if you want to regenerate the core_manifest and core_ep_list files")]`).
+2. Run the unit tests `build.cmd -runTests`.
```

Comment:
>`build.cmd -runTests`. [](start = 22, length = 22)

Running just the one test (it can be run from Visual Studio) might be faster.

Reply:
Can specific tests also be run from the command-line? That could also speed up the process.

Reply:
There is the [mstest library](https://docs.microsoft.com/en-us/previous-versions/ms182489(v=vs.140)?redirectedfrom=MSDN) that supports running C# unit tests through the command-line, but it is no longer maintained in Visual Studio 2019. I think the easiest way is to run the `RegenerateEntryPointCatalog` unit test natively on Visual Studio through its Test Explorer or through shortcuts, which I've now detailed in the developer guide.